onnxruntime/core/providers/cpu/math/element_wise_ops.cc

```diff
@@ -370,6 +370,27 @@ ONNX_CPU_OPERATOR_KERNEL(
     KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<float>()),
     Erf<float>);
 
+ONNX_CPU_OPERATOR_TYPED_KERNEL(
+    Round,
+    11,
+    MLFloat16,
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<MLFloat16>()),
+    Round<MLFloat16>);
+
+ONNX_CPU_OPERATOR_TYPED_KERNEL(
+    Round,
+    11,
+    float,
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<float>()),
```

Comment:
Can these kernels be declared with MayInplace(0, 0)? The implementation would seem to support that.

Reply:
It looks like no `MayImplace` is declared in this file for other similar operators (eg. `Abs`, all trigonometric functions) in this files. Is there any reason for not doing that?

If `Round` is not the only operator to apply `MayImplace`, I would like to add the declarations to all applicable operators in a separated PR.